src/auto-trace.js

```diff
@@ -100,7 +100,9 @@ export function withoutStacktrace(rawError, ...extraMiddlewares){
  */
 function wrapObjectWithError(err, stacktraceErr) {
```

Comment:
Wondering if there's a good way to write jasmine tests for a private method?
(Hopefully something other than separate production/dev builds)